File: JAVA/sum_of_n_natural_number.java
import java.util.Scanner;
public class sum_of_n_natural_number {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        System.out.println("Enter a Number:");
        int n;
        long sum = 0;
        n = s.nextInt();
        long start,stop,duration;
        start = System.nanoTime();
        for ( int i = 1; i <= n; i++){
            sum = sum + i;
        }
        stop = System.nanoTime();
        duration = (stop-start)/1000000;
        System.out.println("Sum of "+n+" natural number is:"+sum);
        System.out.println("Execution time:"+duration+" Milliseconds");
        s.close();
    }
}

File: C++/sum_of_n_natural_number.cpp
#include <iostream>
#include <chrono>

using namespace std;

#define ull long long

int main()
{
    ull n;
    cout << "Enter a Number: ";
    cin >> n;

    auto start = chrono::high_resolution_clock::now();

    ull sum = (n * (n + 1)) / 2;

    auto stop = chrono::high_resolution_clock::now();

    auto duration = chrono::duration_cast<chrono::microseconds>(stop - start).count();

    cout << "Sum of " << n << " natural numbers is: " << sum << endl;
    cout << "Execution time is: " << duration << " Microseconds" << endl;

    return 0;
}
